koku/masu/processor/__init__.py

```diff
@@ -143,3 +143,10 @@ def is_ingress_rate_limiting_disabled():  # pragma: no cover
     if res:
         LOG.info(log_json(msg="ingress rate limiting disabled"))
     return res
+
+
+def override_customer_group_by_limit(account):  # pragma: no cover
+    """Flag the customer to override max group_by count."""
+    account = convert_account(account)
+    context = {"schema": account}
+    return UNLEASH_CLIENT.is_enabled("cost-management.backend.override_customer_group_by_limit", context)
```

Comment:
This function could do the work of determining the limit.
```suggestion
def get_customer_group_by_limit(account: str) -> int:  # pragma: no cover
    """Get the group_by limit for an account."""
    limit = config.GROUP_BY_LIMIT  # this is 2
    account = convert_account(account)
    context = {"schema": account}
    if UNLEASH_CLIENT.is_enabled("cost-management.backend.override_customer_group_by_limit", context):
        limit = config.MAX_GROUP_BY
        
    return limit
```